### DISCUSSION TITLE
Find the mounting point

### DISCUSSION THREAD

**Discussion by EMP83** [nicolargo/glances#2697] [Q&A]
![Glances](https://github.com/nicolargo/glances/assets/36104474/80dbb471-d709-4ef2-82d3-0bdb70ae5fc0)
Is there any way to expand the File sys section to see the mounting point?
I would like to hide it but I can't find it.

Thanks

**Comment by nicolargo** [comment]
You can ask to the Glances API.

In a terminal:

```
$ curl http://0.0.0.0:61208/api/3/fs/device_name
{"device_name":["\/dev\/mapper\/ubuntu--gnome--vg-root","\/dev\/mapper\/ubuntu--gnome--vg-root","\/dev\/mapper\/ubuntu--gnome--vg-root","\/dev\/mapper\/ubuntu--gnome--vg-root"]}
```

**Reply by EMP83** [reply]
Thanks for the reply.
When I run the command, I get this response:
`No such file or directory`
I suspect that this happens because Glances runs in an unRAID OS.

The version of Glances I'm running is:
`glances:latest-full`
